general/mem_manager.hpp

```diff
@@ -600,6 +600,12 @@ class MemoryManager
                          MemoryType h_mt, MemoryType d_mt,
                          bool own, bool alias, unsigned &flags);
 
+   /** @brief If true, class Memory will register the base Memory when creating
+       aliases with Memory<T>::MakeAlias(). */
+   /** Currently, this method returns true when the default device memory type
```

Comment:
Shoudn't `default` be `configured`?

Reply:
Maybe even "the configured default". 😁 

Or do you prefer just "the configured"?